spec.html

```diff
@@ -30865,7 +30890,9 @@ <h1>Static Semantics: StringValue</h1>
             RegExpIdentifierName[?U] RegExpIdentifierPart[?U]
         </emu-grammar>
         <emu-alg>
-          1. Return the String value consisting of the sequence of code units corresponding to |RegExpIdentifierName|. In determining the sequence any occurrences of `\\` |RegExpUnicodeEscapeSequence| are first replaced with the code point represented by the |RegExpUnicodeEscapeSequence| and then the code points of the entire |RegExpIdentifierName| are converted to code units by UTF16Encoding each code point.
+          1. Let _idText_ be the source text matched by |RegExpIdentifierName|.
+          1. Let _idTextUnescaped_ be the result of replacing any occurrences of `\\` |RegExpUnicodeEscapeSequence| in _idText_ with the code point represented by the |RegExpUnicodeEscapeSequence|.
```

Comment:
Same here, we could define CPV for each of the alternatives in RegExpUnicodeEscapeSequence. I would much prefer that to the current text.